src/app/NewCollectionEvent.js

```diff
@@ -195,7 +199,7 @@ define([
 
             $(config.app.header.submitBtn).button('reset');
         },
-        validateReport: function (allowNoFish) {
```

Comment:
is this the cause of #79

Reply:
Of course! I screwed up that refactor. I'll fix it. Thanks for the catch.